### PR TITLE
Created BaselineProfile for app performance improvement support

### LINKED ISSUE #137: Baseline Profile
**Idea Description**
If we introduce the benchmark module, we can make a baseline profile.
Please make it 🙏 

Blocked by
https://github.com/DroidKaigi/conference-app-2022/issues/112

**Reference images and links**
https://developer.android.com/topic/performance/baselineprofiles

### PR DESCRIPTION
**Pull request by hiraok**
## Issue
- close #137 

## Overview (Required)
- Created BaselineProfile, then putted `base-profile.txt` to under `src/main` in `app-android` module 
- Added `semantics` to access composables using UIAutomator for create BaselineProfile
https://developer.android.com/jetpack/compose/testing#uiautomator-interop

## Links
-

### CODE REVIEW

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDayTab.kt [hunk 1]
```diff
@@ -38,7 +41,11 @@ internal fun SessionDayTab(
     Tab(
         selected = selected,
         onClick = { onTabClicked(index) },
-        modifier = Modifier.height(height).padding(end = 8.dp).clip(CircleShape)
+        modifier = Modifier.height(height).padding(end = 8.dp).clip(CircleShape).semantics {
+            if (day == Day2) {
+                testTag = "sessionDayTab"
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDayTab.kt:46]: Thanks!
I think this makes code change a little difficult. Day2 is not only day tab.
I think we don't have to switch tab at the moment. What do you think?

**Comment by mhidaka** [comment]
@hiraok hi 👋 This is just a friendly reminder that Issue is now waiting your comment 😉

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDayTab.kt (hunk 1 above)
Comment (hiraok) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDayTab.kt:46]: I'm sorry. I am late in noticing your comment 🙇 
I didn't understand about other influences.. 
So, remove the `semantics` movement and try again 👍

**Comment by hiraok** [comment]
@takahirom 
I am very sorry for the delay in responding.. 🙇‍♂️ 
Please check to see if there are any problems 🙏

**Comment by takahirom** [comment]
I wouldn't like to have the baseline profile txt to the repository for repository size and want to use the GitHub artifact ideally. So I made the PR that uses your commits, excluding the profile 🙏 
https://github.com/DroidKaigi/conference-app-2022/pull/835

**Comment by takahirom** [comment]
This PR is merged in https://github.com/DroidKaigi/conference-app-2022/pull/835 🙏

**Pull request closed without merging**